src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/MethodologyStatusPage.tsx

```diff
@@ -28,101 +27,92 @@ const statusMap: Dictionary<string> = {
       permissionService.canMarkMethodologyAsDraft(methodologyId),
     ]);
 
     return {
-      summary,
       canApprove,
       canMarkAsDraft,
     };
   }, [methodologyId]);
 
   const handleSubmit = async (values: FormValues) => {
-    if (!model) {
+    if (!currentMethodology) {
       return;
     }
 
     const nextSummary = await methodologyService.updateMethodology(
       methodologyId,
       {
-        ...model.summary,
+        ...currentMethodology,
         ...values,
       },
     );
 
-    setModel({
-      isLoading: false,
-      value: {
-        ...model,
-        summary: {
-          ...nextSummary,
-        },
-      },
-    });
+    onMethodologyChange(nextSummary);
 
     toggleForm.off();
   };
 
-  const isEditable = model?.canApprove || model?.canMarkAsDraft;
+  const isEditable = permissions?.canApprove || permissions?.canMarkAsDraft;
 
   return (
     <>
       <LoadingSpinner loading={isLoading}>
-        {model ? (
+        {currentMethodology ? (
           <>
             {!isEditing ? (
               <>
                 <h2>Sign off</h2>
 
                 <SummaryList>
                   <SummaryListItem term="Status">
-                    <StatusBlock text={statusMap[model.summary.status]} />
+                    <StatusBlock text={statusMap[currentMethodology.status]} />
                   </SummaryListItem>
-                  {model.summary.status === 'Approved' && (
+                  {currentMethodology.status === 'Approved' && (
                     <>
                       <SummaryListItem term="When to publish">
-                        {model.summary.publishingStrategy === 'WithRelease'
+                        {currentMethodology.publishingStrategy === 'WithRelease'
                           ? 'With a specific release'
-                          : model.summary.publishingStrategy}
+                          : currentMethodology.publishingStrategy}
                       </SummaryListItem>
-                      {model.summary.publishingStrategy === 'WithRelease' && (
+                      {currentMethodology.publishingStrategy ===
+                        'WithRelease' && (
                         <SummaryListItem term="Publish with release">
-                          {model.summary.scheduledWithRelease?.title}
+                          {currentMethodology.scheduledWithRelease?.title}
                         </SummaryListItem>
                       )}
                     </>
                   )}
                   <SummaryListItem term="Owning publication">
-                    {model.summary.owningPublication.title}
+                    {currentMethodology.owningPublication.title}
                   </SummaryListItem>
-                  {model.summary.otherPublications &&
-                    model.summary.otherPublications.length > 0 && (
+                  {currentMethodology.otherPublications &&
+                    currentMethodology.otherPublications.length > 0 && (
```

Comment:
Can make this a little more succinct with `currentMethodology.otherPublications?.length > 0`

Reply:
Did you use the optional chaining operator? `otherPublications?.length`

Reply:
Yes, I tried changing it to `{currentMethodology.otherPublications?.length > 0 && (` and it complained that otherPublications is possibly undefined.

Reply:
Ah right, interesting. Feels like a potential bug in TypeScript. I think in more recent versions they added better handling of conditional paths, so this might be something wrong with our version.

It's fine anyway, but we should probably think about upgrading TypeScript and all of other deps in the near future!